mrmustard/physics/bargmann.py

```diff
@@ -12,8 +12,131 @@
+                    
+As for a ``n``-mode Gaussian Channel :math:`\langle \vec{\alpha}|\Psi(|\vec{\gamma}\rangle\langle\vec{\delta}|)|\vec{\beta}`, the variable vector denotes :math:`\vec{\alpha} = (\alpha^*_0, \alpha^*_1, ..., \alpha^*_{n-1}, \beta_0, \beta_1,..., \beta_{n-1}, \delta^*_0, \delta^*_1, ..., \delta^*_{n-1}, \gamma_0, \gamma_1,..., \gamma_{n-1})`.
+
+    .. code-block::
+
+                             ╔═══════╗
+        delta^*_0      ─────▶║       ║─────▶ alpha^*_0
+        delta^*_1      ─────▶║       ║─────▶ alpha^*_1
+                          ...║       ║...
+        delta^*_(n-1)  ─────▶║  Phi  ║─────▶ alpha^*_(n-1)    
+        gamma_0        ─────▶║       ║─────▶ beta_0
+        gamma_1        ─────▶║       ║─────▶ beta_1
+                          ...║       ║...
+        gamma_(n-1)    ─────▶║       ║─────▶ beta_(n-1)  
+                             ╚═══════╝    
+
+The wires in the diagram below correspond to the `out_bra` wires (:math:`\alpha^*`), the `in_bra` wires (:math:`\delta^*`), `out_ket` wires (:math:`\beta`) and the `in_ket` wires (:math:`\gamma`) in :class:`~mrmustard.lab_dev.wires.Wires`.
+
+The computation of quantum circuits with Bargmann representation can be considered as the inner product of two Bargmann representations (which can be realized by Gaussian integrals for all Gaussian objects computation), such as applying the unitary on a state, contracting two unitaries, applying the channel on a state, and etc.
+
+For example, applying a single-mode unitary :math:`U` on a single-mode pure state :math:`|\psi\rangle` is to multiply the Bargmann representation of the unitary and the state and then to integral the variables on the common wire between then:
+
+    .. math::
+        U|\psi\rangle = \int d^2 \alpha |\beta\rangle \langle\beta|U|\alpha\rangle \langle\alpha|\psi\rangle.
+
+    .. code-block::
+
+        ╔═══════╗                                ╔═════╗
+        ║ |psi> ║─────▶ alpha^*_0   alpha_0─────▶║  U  ║─────▶ beta^*_0
+        ╚═══════╝                                ╚═════╝
+            |
+            | integral on alpha
+            |
+        ╔════════╗
+        ║ |psi'> ║─────▶ beta^*_0
+        ╚════════╝
+
+Another example, applying a single-mode Gaussian channel :math:`\Phi` on a single-mode pure state, one needs to add the ``adjoint`` of the state and contract with the input of the channel:
+
+    .. code-block::
+
+                                                 ╔═══════╗
+                                    alpha_0─────▶║       ║─────▶ beta_0
+                                                 ║  Phi  ║
+        ╔═══════╗                                ║       ║
+        ║ |psi> ║─────▶ alpha^*_0 alpha^*_0─────▶║       ║─────▶ beta^*_0
+        ╚═══════╝                                ╚═══════╝ 
+            |
+            | Add the ajoint part
```

Comment:
if you mean this is "equality" or "follows", it's very confusing to me because it looks like a wire